### PR TITLE
firmware log parser added

### PR DESCRIPTION
**Pull request by Tamir91**
Tracked on [LRS-789]

### CODE REVIEW

File: scripts/firmware_log_parser.py [hunk 1]
```diff
@@ -0,0 +1,513 @@
+#!/usr/bin/env python3
+"""
+This script parse firmware log
```
Comment (Nir-Az) [inline scripts/firmware_log_parser.py:3]: Please add a short readme

Comment (Nir-Az) [reply scripts/firmware_log_parser.py:3]: inside a new folder called `fw_log_parser` next to the parser

Comment (Tamir91) [reply scripts/firmware_log_parser.py:3]: May I add the `HWLoggerEventsDS5.xml` file to the folder because I wrote about him in a readme?

Comment (Nir-Az) [reply scripts/firmware_log_parser.py:3]: No need, it's from the FW team

**Review by dmipx — APPROVED** [review]
Good job!

**Pull request merged**